docs/source/rbac/scopedict.py

```diff
@@ -0,0 +1,97 @@
+"""Scope definitions"""
+
+
+def get_scope_dict():
+    """Returns a nested dictionary of all available scopes:
+
+    {scopename: {'description': description,
+                 'subscopes': [immediate subscopes]},
+    }
+
+    without 'subscopes' key if the scope has no subscopes.
+    """
+    scope_dict = {
```

Comment:
Once added to jupyterhub.scopes.py, this should probably be a module-level dict, rather than a new dict created on each call, e.g. `jupyterhub.scopes.scope_definitions = { scopename: ...}`

Reply:
Good idea, thanks.